examples/bridge-domain/bridge/cmd/composite.go

```diff
@@ -79,12 +91,24 @@ func newVppAgentBridgeComposite(configuration *common.NSConfiguration) *vppAgent
 	}
 	configuration.CompleteNSConfiguration()
 
-	logrus.Infof("vppAgentBridgeComposite")
+	bridgeDomain := &l2.BridgeDomain{
+		Name:                "brd",
+		Flood:               true,
+		UnknownUnicastFlood: true,
+		Forward:             true,
+		Learn:               true,
+		MacAge:              120,
+		Interfaces:          make([]*l2.BridgeDomain_Interface, 0),
+	}
 
 	newVppAgentBridgeComposite := &vppAgentBridgeComposite{
```

Comment:
This is excellent work.. would you consider adding it to the SDK so folks could use it as a reusable component for building their own NSEs?  This is quite a bit better than our existing [bridge component](https://github.com/networkservicemesh/networkservicemesh/blob/master/sdk/vppagent/bridge.go) in the SDK :)